Vungle/src/main/java/com/applovin/mediation/adapters/VungleInitializer.java

```diff
@@ -0,0 +1,71 @@
+package com.applovin.mediation.adapters;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import com.vungle.ads.InitializationListener;
+import com.vungle.ads.VungleAds;
+import com.vungle.ads.VungleError;
+import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class VungleInitializer implements InitializationListener {
+
+  private static final VungleInitializer instance = new VungleInitializer();
+  private final AtomicBoolean isInitializing = new AtomicBoolean(false);
+  private final ArrayList<VungleInitializationListener> initListeners;
+
+  @NonNull
+  public static VungleInitializer getInstance() {
+    return instance;
+  }
+
+  private VungleInitializer() {
+    initListeners = new ArrayList<>();
+    VungleAds.setIntegrationName( VungleAds.WrapperFramework.max,
+        com.applovin.mediation.adapters.vungle.BuildConfig.VERSION_NAME );
+  }
+
+  public void initialize(
+      final @NonNull String appId,
+      final @NonNull Context context,
+      @NonNull VungleInitializationListener listener) {
+
+    if (VungleAds.isInitialized()) {
+      listener.onInitializeSuccess();
+      return;
+    }
+
+    if (isInitializing.getAndSet(true)) {
+      initListeners.add(listener);
```

Comment:
does MAX have automatic styling for them? in iOS, they require us to do ( variable ) like Android, but we need to do it manually...

Reply:
Android IDE support automatic code format. I created the code format styles for AdMob and MAX:
<img width="301" alt="image" src="https://github.com/Vungle/AppLovin-MAX-SDK-Android/assets/46985415/4304c838-cead-483b-a6a0-049f13b40362">